doc/api/vm.md

```diff
@@ -311,6 +311,11 @@ console.log(util.inspect(sandbox));
 ## vm.runInDebugContext(code)
 <!-- YAML
 added: v0.11.14
+deprecated: v8.0.0
```

Comment:
This change should go in separately with v8.x.

Reply:
@TimothyGu Done (but `deprecated: v8.0.0` is correct since that’s when we docs-deprecated)